db/controllers/categories.controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchCategories } = require("../models/categories.model");
+
+exports.getAllCategories = (request, response) => {
+  fetchCategories()
+    .then((categories) => {
+      response.status(200).send(categories);
+    })
+    .catch((err) => {
+      next(err);
```

Comment:
We want to ensure we add a path to handle the 404 and it would 
also be appropriate to include a 500 error handler at this stage.
:)

Reply:
@HannahBurks thanks for feedback, changed made and pushed